### PR TITLE
Update legal name of Donghee Na

### PR DESCRIPTION
**Pull request by corona10**
Hi, my English legal name was updated from "Dong-hee Na" to "Donghee Na"
There were several issues with my legal English name and I got approval from Korea government about updating it, 
so I would like to request updating my legal name in this repo too :)

Related story: https://x.com/dongheena92/status/1702490849282810223?s=20

### CODE REVIEW

**Review by hartwork — COMMENTED** [review]
Hi @corona10,

I believe there may be two more places where this change should be applied:

#### 1. The change log

```console
# git --no-pager grep -n Dong-hee
expat/Changes:329:            Dong-hee Na
```

#### 2. Git `.mailmap` file

…at https://github.com/libexpat/libexpat/blob/master/.mailmap

```console
# git log | grep "Author: Dong-hee"
Author: Dong-hee Na <donghee.na@python.org>
```

What do you think?

**Comment by corona10** [comment]
@hartwork Thank you for catching this :)

File: .mailmap [hunk 1]
```diff
@@ -17,3 +17,4 @@ Steven Solie <steven@solie.ca> <ssolie@users.sourceforge.net>
 Thomas Beutlich <tc@tbeu.de>
 Viktor Szakats <commit@vsz.me> <vszakats@users.noreply.github.com>
 Zhongyuan Zhou <zhouzhongyuan@huawei.com>
+Donghee Na <donghee.na@python.org>
```
Comment (hartwork) [inline .mailmap:20]: While the entry looks good and seems to work well in practice, previously the lines in this file were sorted alphabetically. Could you move this line after `Ben Wagner` please?

File: expat/Changes [hunk 2]
```diff
@@ -1114,7 +1114,7 @@ Release 2.1.0 Sat March 24 2012
                 See bug #2990652 and patch #3312568.
           Added run-benchmark target to Makefile.in - relies on testdata module
                 present in the same relative location as in the repository.
-          
+
```
Comment (hartwork) [inline expat/Changes:1117]: Please revert the unrelated changes in white space to this file, thanks!

**Review by hartwork — CHANGES_REQUESTED** [review]
@corona10 we're almost there, please see the two comments below:

**Comment by corona10** [comment]
@hartwork Done thanks!

File: expat/Changes [hunk 3]
```diff
@@ -1088,7 +1088,7 @@ Release 2.1.0 Sat March 24 2012
         - Bug Fixes:
           #1742315: Harmful XML_ParserCreateNS suggestion.
           #1785430: Expat build fails on linux-amd64 with gcc version>=4.1 -O3.
-          #1983953, 2517952, 2517962, 2649838: 
+          #1983953, 2517952, 2517962, 2649838:
```
Comment (hartwork) [inline expat/Changes:1091]: @corona10 there is one more unrelated whitespace change here^^

**Review by hartwork — APPROVED** [review]
@corona10 thanks for the adjustments!

**Pull request merged**